src/test/java/bio/terra/fixtures/ResourceFixtures.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.fixtures;
+
+import bio.terra.metadata.BillingProfile;
+
+import java.util.Collections;
+import java.util.Random;
+import java.util.stream.Collectors;
+
+public final class ResourceFixtures {
+    private ResourceFixtures() {}
+
+    public static String randomHex(int n) {
+        Random r = new Random();
+        StringBuffer sb = new StringBuffer();
+        while (sb.length() < n) {
+            sb.append(Integer.toHexString(r.nextInt(16)));
+        }
+        return sb.toString();
+    }
+
+    // Google billing account ids are three sets of six capitalized hex characters separated by dashes
+    public static String randomBillingAccountId() {
+        return Collections.nCopies(3, 6)
+            .stream()
+            .map(ResourceFixtures::randomHex)
+            .collect(Collectors.joining("-")).toUpperCase();
```

Comment:
Wow, that is pretty obscure to concatenate three strings.

Reply:
Ugh yeah, I was trying to reproduce the pythonic way to do this which is very readable. I'll switch to the slightly longer, much less obscure version.